ui/tract-renderer/src/main/java/org/cru/godtools/tract/widget/PageContentLayout.java

```diff
@@ -82,30 +99,38 @@ public void onAnimationEnd(final Animator animation) {
         }
     };
 
+    //region Initialization
     public PageContentLayout(@NonNull final Context context) {
         this(context, null);
+        initialization(context);
     }
 
     public PageContentLayout(@NonNull final Context context, @Nullable final AttributeSet attrs) {
         this(context, attrs, 0);
+        initialization(context);
     }
 
     public PageContentLayout(@NonNull final Context context, @Nullable final AttributeSet attrs,
                              final int defStyleAttr) {
         super(context, attrs, defStyleAttr);
+        initialization(context);
+    }
+
+    private void initialization(@NonNull Context context) {
```

Comment:
I think I would rather not have an `initialization()` method and be able to keep the singletons you are using as `final`.

Reply:
OK I will switch it back I was trying to prevent duplicate code.

Reply:
it's a toss up, I like `final` because it makes the values immutable, which I value over a few lines of duplicated code.